astropy/coordinates/earth.py

```diff
@@ -574,9 +574,15 @@ def _get_site_registry(cls, force_download=False, force_builtin=False):
             if force_download or not cls._site_registry:
                 try:
                     if isinstance(force_download, str):
-                        cls._site_registry = get_downloaded_sites(force_download)
+                        cls._site_registry = get_downloaded_sites(
+                            jsonurl=force_download, cache="update"
+                        )
                     else:
-                        cls._site_registry = get_downloaded_sites()
+                        if force_download:
+                            cache = "update"
+                        else:
+                            cache = True
+                        cls._site_registry = get_downloaded_sites(cache=cache)
```

Comment:
```suggestion
                        cls._site_registry = get_downloaded_sites(cache="update")
```
If I'm not mistaken the only way to get here if `force_download` is _not_ set is that the cache is not yet loaded, so there should be no difference between `True` and `"update"`?